client-app/src/Components/DonatedItemsList.js

```diff
@@ -173,11 +174,11 @@ function DonatedItemsList() {
                 <option value="" disabled selected>
                   Filter by Status
                 </option>
-                <option value="Donated">Donated</option>
-                <option value="In Storage Facility">In Storage Facility</option>
-                <option value="Refurbished">Refurbished</option>
-                <option value="Received">Received</option>
-                <option value="Item Sold">Item Sold</option>
+                <option value={ItemStatus.DONATED}>Donated</option>
```

Comment:
This enums increased abstraction, such that user may not make mistakes again while entering the data.